### PR TITLE
fix: /image/{name or id}/json returns RootFS layers

### PR DESCRIPTION
**Pull request by matejvasek**
Signed-off-by: Matej Vasek <mvasek@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
/approve
LGTM
@jwhonce @baude @vrothberg PTAL

**Comment by rhatdan** [comment]
Thanks @matejvasek 
/lgtm
/hold

**Comment by jwhonce** [comment]
LGTM

**Comment by matejvasek** [comment]
I just added some nil checks, `info.RootFS` should be always non-`nil`, not sure about `info.RootFS.Layers`. But better save than sorry.

File: pkg/api/handlers/types.go [hunk 1]
```diff
@@ -271,6 +271,15 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 	if err != nil {
 		return nil, err
 	}
+
+	rootfs := docker.RootFS{}
+	if info.RootFS != nil && info.RootFS.Layers != nil {
```
Comment (TomSweeneyRedHat) [inline pkg/api/handlers/types.go:276]: Should this be split?  i.e. test RootFS for nil and then set RootFS.Type if it is not nil, then test for RootFS.Layers and set that if it is not nil.

Comment (matejvasek) [reply pkg/api/handlers/types.go]: I fix that. It turned out that it's OK to call `len()` and `range` on nil slice.

**Comment by rhatdan** [comment]
Thanks @matejvasek 
/lgtm
/hold
/approve

**Pull request merged**